### DISCUSSION TITLE
How to get current count of cargo items onboard of a vehicle

### DISCUSSION THREAD

**Discussion by odashevsky** [google/or-tools#3652] [Routing (and legacy CP) questions]
Assuming that I have a solution to a VRP problem, is there a or-tools API call that will tell me how many cargo items are currently onboard of a vehicle at certain location? Clearly, I can trace it myself when processing the solution by counting pickups/dropoffs when going through the vehicles trip, but it would be easier if there was simply a call that will tell me "for vehicle number A at its node index B, load is 4 packages" or something similar. Thanks.

**Comment by lperron** [comment]
get the value of the cumul var from the solution.

**Reply by Mizux** [reply]
As laurent said if you have a capacity/load dimension you can get the value of the CumulVar at each location...
see: vrp_token where we retrieve value of the CumulVar in the solution printer...
https://github.com/google/or-tools/blob/f460e9b0fcd444c37878ec64be9822d40fb375f4/ortools/constraint_solver/samples/vrp_tokens.py#L57
https://github.com/google/or-tools/blob/f460e9b0fcd444c37878ec64be9822d40fb375f4/ortools/constraint_solver/samples/vrp_tokens.py#L68-L70

**Comment by odashevsky** [comment]
Thank you!
